automated-comments/javascript/gigasecond/export_inline.md

```diff
@@ -1,7 +1,5 @@
 Functions, classes and constants can be `export`ed inline.
 
 ```javascript
-const %{const.name} = (...)
-
 export %{method-signature}
```

Comment:
```suggestion
export %{method-signature}
```

The comments talk about exporting functions, classes and constants, but the example mentions a method-signature. Should that maybe be someting like `%{exported-identifier}` or something like that?